### PR TITLE
GH-13291: Add unit tests to ChannelGroupEnableCmdF mmctl command

### PR DESCRIPTION
**Pull request by lurcio**
#### Summary
Add unit tests to the "ChannelGroupEnableCmdF" mmctl command.

This is a retry of this PR https://github.com/mattermost/mmctl/pull/74 - I encountered a weird syncing issue and thought it was cleaner to start from scratch.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/13291

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @lurcio! Heads up that reviews may take longer with the holidays :)

File: commands/group_test.go [hunk 1]
```diff
@@ -166,3 +166,303 @@ func (s *MmctlUnitTestSuite) TestTeamGroupEnableCmd() {
 		s.Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestChannelGroupEnableCmdF() {
+	s.Run("Enable group constrains with existing team and channel", func() {
```
Comment (streamer45) [inline commands/group_test.go]: Small typo here and in the cases below, should be `constraints`.

File: commands/group_test.go [hunk 2]
```diff
@@ -166,3 +166,303 @@ func (s *MmctlUnitTestSuite) TestTeamGroupEnableCmd() {
...
+			Times(1)
+
+		err := channelGroupEnableCmdF(s.client, &cobra.Command{}, []string{channelArg})
+		s.Require().NotNil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.EqualError(err, "Channel '"+channelArg+"' has no groups associated. It cannot be group-constrained")
+	})
+
+	s.Run("Enable group constrains with nonexistant team", func() {
```
Comment (streamer45) [inline commands/group_test.go]: Small typo here and in the test case below, should be `nonexistent`.

File: commands/group_test.go (hunk 1 above)
Comment (lurcio) [reply commands/group_test.go]: Thank you for the feedback. This has been corrected now.

File: commands/group_test.go (hunk 2 above)
Comment (lurcio) [reply commands/group_test.go]: Thank you for the feedback. This has been corrected now.

**Comment by agnivade** [comment]
I still don't see where `Enable group constraints with GetChannelByNameIncludeDeleted error` returns an error. Both GetChannelByNameIncludeDeleted and GetChannel do not return any error AFAI can see.

**Comment by lurcio** [comment]
Hi @agnivade. Sorry, I have changed GetChannelByNameIncludeDeleted to return an error now. Is that right?

**Pull request merged**